planning/scenario_planning/lane_driving/behavior_planning/behavior_velocity_planner/src/scene_module/intersection/scene_intersection.cpp

```diff
@@ -256,12 +264,18 @@ bool IntersectionModule::checkCollision(
   bool collision_detected = false;
   for (const auto & object : target_objects.objects) {
     bool has_collision = false;
-    for (const auto & predicted_path : object.state.predicted_paths) {
+    for (const auto & predicted_path : object.kinematics.predicted_paths) {
+      // TODO(murooka) confidence or existence_probability
       if (predicted_path.confidence < planner_param_.min_predicted_path_confidence) {
         // ignore the predicted path with too low confidence
         continue;
       }
```

Comment:
Which should I use? conficence or existence_probability? @yukkysaito

Reply:
then remove TODO(murooka)

Reply:
you should use confidence
https://github.com/tier4/autoware_auto_msgs/blob/tier4/main/autoware_auto_perception_msgs/msg/PredictedPath.idl#L12

Reply:
https://github.com/tier4/AutowareArchitectureProposal.iv/pull/605/commits/7d93a0fd4694593dae9fef814f347ebfeef2ba7a